uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/WorkPatternTest.php

```diff
@@ -204,6 +204,25 @@ public function testGetValuesArrayShouldReturnEmptyArrayWhenWorkPatternDoesntExi
         $this->assertEmpty($values);
     }
 
+    /**
+     * @expectedException CRM_HRLeaveAndAbsences_Exception_InvalidWorkPatternException
+     * @expectedExceptionMessage You cannot disable a Work Pattern if it's the last one
+     */
+    public function testCannotDisabledPatternIfItIsTheLastWorkPatternEnabled() {
```

Comment:
change testCannotDisabledPatternIfItIsTheLastWorkPatternEnabled

to 

testCannotDisablePatternIfItIsTheLastWorkPatternEnabled ... just remove the d from disabled :D ... the same in the other test method